.github/rdmo-app-ci-script.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+git clone https://$GITHUB_TOKEN@github.com/rdmorganiser/rdmo.git
+# need rdmo only for testing/config/settings
+git clone https://$GITHUB_TOKEN@github.com/rdmorganiser/rdmo-app.git
+
+cd rdmo-app
```

Comment:
I suggest to add "|| exit 1" to make sure that this script does not run further in case the folder does not exist.